### ISSUE TITLE
Entry for "Erdbeben" in 2_H's ToC

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/faust-web#613]
https://www.faustedition.net/document?sigil=2_H&page=1&view=print&section=#d19367
shows "Erdbeben" as separate ToC entry:

![grafik](https://github.com/user-attachments/assets/7acd2134-23d1-4cfb-90fb-0ff242927872)

I cannot explain why this is the case, since the textual transcript has no `div` start at this point. @thvitt, can you explain this?

**Comment by thvitt** [comment]
It’s the toc entry for div 2.2.3.3

https://github.com/faustedition/faust-xml/blob/d434de9461d62a2c5d7ef112dc126dc1f2e845ca/xml/transcript/gsa/391098/391098.xml#L7908

We extract a heading for that div by finding a source element using this rule:

https://github.com/faustedition/faust-gen-html/blob/be7fdaa2e972ed37f75de552f7fd84b64cd389bc/xslt/add-metadata.xsl#L300-L302

`(head, stage, sp, *)[1]` means "the first head, if any, otherwise the first stage, if any, otherwise the first sp, if any, otherwise the first element". If you look at the TEI source, there is no head for that scene but a sp _before_ the first stage, but the rule above prefers the stage over the sp.

**Comment by gerritbruening** [comment]
How do you get from

https://github.com/faustedition/faust-xml/blob/d434de9461d62a2c5d7ef112dc126dc1f2e845ca/xml/transcript/gsa/391098/391098.xml#L7908

to the ["Erdbeben" `stage`](https://github.com/faustedition/faust-xml/blob/d434de9461d62a2c5d7ef112dc126dc1f2e845ca/xml/transcript/gsa/391098/391098.xml#L7947)? It seems that the first `sp` is skipped. What makes thinks worse is that `stage[@xml:id='before_7495_a']` is needed here as result of a `transpose`.

**Comment by thvitt** [comment]
Yes, we're looking for `(head, stage, sp, *)[1]` that are children of the div. The sp has lower priority than the stage in this rule, and the stage inside the sp is not seen. 

We could probably modify this to sth. like (head, descendant::stage, sp, *)[1] to find the stage inside sp or to `(head, (stage | sp), *)` to treat stage and sp equally (both might have consequences elsewhere). What is it that you would like to see?

**Comment by gerritbruening** [comment]
In this case, as well as in general, it would make sense to start from the emended stage because only there is it clear what should be considered the initial chunk-level element of a `div`. It might be that this only makes a real difference in this specific case, so a special rule referring to the specific `xml:id` could also be considered. However, I would find it more logical to generally start from the emended stage. I cannot assess, though, how complicated this adjustment would be.
